submissions/webdevtoday/html-css-popup/index.html

```diff
@@ -8,105 +8,152 @@
-                            <img src="popup-icons/apps.png" alt="apps icon" class="nav-popup__icon">
-                            <span class="nav-popup__title">Apps</span>
-                        </a>
-                        <a href="#document" class="nav-popup__item text-decoration-none">
-                            <img src="popup-icons/documents.png" alt="documents icon" class="nav-popup__icon nav-popup__icon--documents">
-                            <span class="nav-popup__title nav-popup__title--documents">Documents</span>
-                        </a>
-
-                        <input type="checkbox" class="nav-popup__state-flag" id="nav-popup__state-flag">
-                        <label for="nav-popup__state-flag" class="nav-popup__item nav-popup__more">More</label>
-                        <div class="nav-popup__container nav-popup__container--more">
-                            <a href="#chat" class="nav-popup__item text-decoration-none">
-                                <img src="popup-icons/chat.png" alt="chat icon" class="nav-popup__icon">
-                                <span class="nav-popup__title">Chat</span>
-                            </a>
-                            <a href="#classroom" class="nav-popup__item text-decoration-none">
-                                <img src="popup-icons/classroom.png" alt="classroom icon" class="nav-popup__icon">
-                                <span class="nav-popup__title">Classroom</span>
-                            </a>
-                            <a href="#collection" class="nav-popup__item text-decoration-none">
-                                <img src="popup-icons/collection.png" alt="collection icon" class="nav-popup__icon">
-                                <span class="nav-popup__title">Collection</span>
-                            </a>
-                            <a href="#mars" class="nav-popup__item text-decoration-none">
-                                <img src="popup-icons/mars.png" alt="mars icon" class="nav-popup__icon">
-                                <span class="nav-popup__title">Mars</span>
-                            </a>
-                            <a href="#blogs" class="nav-popup__item text-decoration-none">
-                                <img src="popup-icons/blogs.png" alt="blogs icon" class="nav-popup__icon">
-                                <span class="nav-popup__title">Blogs</span>
-                            </a>
-                            <a href="#promos" class="nav-popup__item text-decoration-none">
-                                <img src="popup-icons/promo.png" alt="promos icon" class="nav-popup__icon">
-                                <span class="nav-popup__title">Promos</span>
-                            </a>
-                            <a href="#Kottans" class="nav-popup__item text-decoration-none kottans-link">Even more from Kottans</a>
-                        </div>
+                    <div class="main-nav__popup nav-popup">
```

Comment:
The first part of popup inside the checkbox label, and at the same time the second part of popup outside of the checkbox label, which approach is preferable?

Reply:
Yes, the first part is an absolute element, while the second part is static flow of html code.

Reply:
When I hovered on the popup icon menu icon have hover too
![image](https://user-images.githubusercontent.com/16196199/97757803-2fc31580-1b06-11eb-8ca3-002b4fe752c0.png)

Reply:
I thought it must be so. Or not?

Reply:
Ok, I've fixed nested div in label and hover effect.